.gitignore

```diff
@@ -23,3 +23,6 @@ html/
 bindings/py/nupic/bindings/*.cxx
 bindings/py/nupic/bindings/*.py
 bindings/py/nupic/bindings/*.so
+
+# examples' output
+src/examples/algorithms/anomaly_out.csv
```

Comment:
Remove this. We don't want to leave misc. files lying around. If an example generates a file that we **do** want to leave around that is fine but we shouldn't put it in here or it will be hard for users to know it is there since it won't show up in `git status`.